src/_gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -100,8 +100,8 @@ vorsorge2004_vorwegabzug:
   description:
     de: >-
       § 10 (4a) EStG. Früher §10 (3) EStG. Hier steht jeweils der Wert für den
-      Steuerpflichtigen. Wird verdoppelt für gemeinsam veranlagte Ehegatten. Der gesamte
```

Comment:
Can you revert, please? Else this will just be going back and forth down the road...